functions/.eslintrc.js

```diff
@@ -20,13 +20,11 @@ module.exports = {
   ignorePatterns: [
     "/lib/**/*", // Ignore built files.
   ],
-  plugins: [
-    "@typescript-eslint",
-    "import",
-  ],
+  plugins: ["@typescript-eslint", "import"],
```

Comment:
please, revert this. we are using the indentation on multiline on lists everywhere in the project